snap/export_test.go

```diff
@@ -19,12 +19,27 @@
 
 package snap
 
+import "github.com/snapcore/snapd/release"
+
 var (
-	ImplicitSlotsForTests        = implicitSlots
-	ImplicitClassicSlotsForTests = implicitClassicSlots
-	NewHookType                  = newHookType
+	NewHookType = newHookType
 )
 
+func ImplicitSlotsForTests() []string {
+	result := implicitSlots
+	// fuse-support is disabled on trusty due to usage of fuse requiring access to mount.
+	// we do not want to widen the apparmor profile defined in fuse-support to support trusty
+	// right now.
+	if !(release.ReleaseInfo.ID == "ubuntu" && release.ReleaseInfo.VersionID == "14.04") {
```

Comment:
I don't think we need to do that here, otherwise we're just testing the test logic itself. We can probably revert the changes to this file completely.